src/cf-layout/src/cf-layout.less

```diff
@@ -675,8 +675,10 @@
 
         margin-bottom: unit(@grid_gutter-width / 22px, em);
 
-        .respond-to-max(800px, {
-            .heading-4();
+        .respond-to-max(@bp-sm-max, {
+            // Use the same regular weight but reduce the sizes to h4 size
+            // Can't use .into-paragraph because it has a different breakpoint
+            font-size: unit( 18px / @base-font-size-px, em );
```

Comment:
Or there. Nevermind. The only thing needing doing is adjusting the margin-bottom.